src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -90,10 +89,13 @@ def register_options(cls, register, register_jvm_tool):
                   'For example, 1/3 means run tests number 2, 5, 8, 11, ...')
     register('--suppress-output', action='store_true', default=True,
              help='Redirect test output to files in .pants.d/test/junit.')
-    # TODO(gmalmquist): Use the build root instead of the path of the first target by default.
     register('--cwd', default=_CWD_NOT_PRESENT, nargs='?',
-             help='Set the working directory. If no argument is passed, use the first target path. '
+             help='Set the working directory. If no argument is passed, use the build root. '
```

Comment:
If the default were None, then setting `--cwd` on the command line with no argument would have the same effect as not putting --cwd on the command line.

Now, if you specify `--cwd`, the value is None, which means do the same as what you have now listed as: `--cwd-default-to-first-target`  The only use case we had for this behavior is specific to maven layout and our practice of puttting test targets in a BUILD file at the maven project level directory.

I don't think we should say anything about the default behavior. If you don't set --cwd you get the current working directory. If you want to set cwd to the build root you can always do that with an explicit cwd directory.

We don't have to address everything on github.  Why don't you can go ahead an post on RBCommons so more reviewers will see it.